### PR TITLE
feat: add new displayName metadata

### PR DESCRIPTION
**Pull request by MartinGbz**

### CODE REVIEW

File: src/infrastructure/group-store/dynamodb-group-store.test.ts [hunk 1]
```diff
@@ -50,6 +50,17 @@ describe("test groups stores", () => {
     const nameHash3 = keccak256(toUtf8Bytes(savedGroup3.name + "/" + "2"));
     const savedId3 = BigNumber.from(nameHash3).mod(UINT128_MAX).toHexString();
     expect(savedGroup3.id).toBe(savedId3);
+
+    const savedGroup4 = await dynamodbGroupStore.save(testGroups.group1_2);
```
Comment (yum0e) [inline src/infrastructure/group-store/dynamodb-group-store.test.ts:54]: What is the aim of this test?
There should be a check after the update correct? Why not update the displayName?

File: src/infrastructure/group-store/group-store.test.ts [hunk 2]
```diff
@@ -52,6 +52,17 @@ describe("test groups stores", () => {
     const nameHash3 = keccak256(toUtf8Bytes(savedGroup3.name + "/" + "2"));
     const savedId3 = BigNumber.from(nameHash3).mod(UINT128_MAX).toHexString();
     expect(savedGroup3.id).toBe(savedId3);
+
+    const savedGroup4 = await groupStore.save(testGroups.group1_2);
```
Comment (yum0e) [inline src/infrastructure/group-store/group-store.test.ts:56]: same here what is the purpose of the test?

File: src/topics/group/group.api.schema.ts [hunk 3]
```diff
@@ -18,8 +18,13 @@ const group = {
     },
     name: {
       type: "string",
-      description: "Badge name",
-      example: "Example Badge",
+      description: "group name",
+      example: "example-group",
+    },
+    displayName: {
```
Comment (yum0e) [inline src/topics/group/group.api.schema.ts:24]: Can we have a test for the schema somewhere?

File: src/infrastructure/group-store/dynamodb-group-store.test.ts (hunk 1 above)
Comment (MartinGbz) [reply src/infrastructure/group-store/dynamodb-group-store.test.ts:54]: The purpose of this test is to make sure that a group with a displayName field can be saved &updated in the dynamoDB without any errors. But I agree, for the update we can also update the displayName and check that all field has been successfully updated  👍

File: src/infrastructure/group-store/group-store.test.ts (hunk 2 above)
Comment (MartinGbz) [reply src/infrastructure/group-store/group-store.test.ts:56]: Same than this answer: https://github.com/sismo-core/sismo-hub/pull/2027/files#r1273486184

**Review by yum0e — APPROVED** [review]

**Pull request merged**